src/python/pants/testutil/file_test_util.py

```diff
@@ -1,64 +0,0 @@
-# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
-import os
-
-
-def exact_files(directory, ignore_links=False):
```

Comment:
I think these suffered from a discoverability problem. We weren't ever using them. Instead, inlining the logic has seemed to suffice.